frontend/benefit/applicant/src/components/applications/forms/application/step2/useApplicationFormStep2.ts

```diff
@@ -134,10 +136,17 @@ const useApplicationFormStep2 = (
     });
   };
 
-  const handleSubmitNext = (): void => handleSubmit(3);
-
-  const handleSubmitBack = (): void =>
-    setApplicationTempData({ ...applicationTempData, currentStep: 1 });
+  const handleSubmitBack = (): void => {
+    setStep(1);
+    const currentApplicationData: ApplicationData = snakecaseKeys(
```

Comment:
Why snakecase is needed? Or could we use snakecase also in frontend to prevent these conversions?

Reply:
for consistency, i don't see a problem in conversion

Reply:
basically, in my opinion, if we use everywhere else in react app the camelCase for props, we should not rely on django app conventions. We could have 10 integrations in different languages with different conventions, and its fine to map it in your application to common form.